src/program/lwaftr/setup.lua

```diff
@@ -324,6 +324,9 @@ function load_on_a_stick(c, conf, args)
       link_source(c, v4v6..'.v4', v4v6..'.v6')
       link_sink(c, v4v6..'.v4', v4v6..'.v6')
    else
+      assert(queue.external_interface.mac ~= queue.internal_interface.mac,
```

Comment:
Why is this requirement here?  Does it not work to have the same MAC address but different VLAN tags?

Reply:
Apparently it doesn't work. I saw 100% packets dropped with that configuration. Then it occurred to me to change the MAC addresses (and the data samples) and packets get routed.

I researched a bit to know the actual reason and I found something. Here's an excerpt from the Intel VMDq white paper that acknowledges this limitation:

```
VMDq has certain limitations on sorting packets based on VLAN tags.  VMDq only sorts packets 
based on MAC address or VLAN tags. VMDq can not differentiate between two packets with the same MAC address and different VLAN tags.
```

https://www.intel.com/content/dam/www/public/us/en/documents/white-papers/vmdq-technology-paper.pdf

Reply:
Good find and great assert, then!